samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -25,6 +25,12 @@
     /* "Key": // dotnet user-secrets set "Embedding:Key" "MY_EMBEDDING_KEY" */
   },
 
+  "AzureSpeechConfig": {
```

Comment:
To follow the other conventions in this file, I recommend renaming this section to "AzureSpeech"